src/components/HomeSection.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import { QueryHookOptions, useQuery } from 'react-apollo';
+import { ActivityIndicator, View } from 'react-native';
+import { NavigationScreenProp } from 'react-navigation';
+
+import { colors, consts, device } from '../config';
+import { parseListItemsFromQuery } from '../helpers';
+import { useHomeRefresh } from '../hooks/HomeRefresh';
+import { getQuery } from '../queries';
+import { SettingsContext } from '../SettingsProvider';
+import { Button } from './Button';
+import { ListComponent } from './ListComponent';
+import { LoadingContainer } from './LoadingContainer';
+import { Title, TitleContainer, TitleShadow } from './Title';
+import { Touchable } from './Touchable';
+import { Wrapper } from './Wrapper';
+
+type Props = {
+  categoryTitle: string;
+  categoryTitleDetail?: string;
+  categoryButton: string;
+  fetchPolicy:
+    | 'cache-first'
+    | 'network-only'
+    | 'cache-only'
+    | 'no-cache'
+    | 'standby'
```

Comment:
where do you have these strings from? `standby` is new for me. i do not think that our apollo version does already know that

Reply:
![Screenshot 2021-02-08 at 14 00 30](https://user-images.githubusercontent.com/59824597/107223695-ca40c800-6a16-11eb-944d-27a5da8eee02.png)

I took them from the prop types that the useQuery hook expects, so that hook should support it.